fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java

```diff
@@ -226,6 +233,17 @@ private List<Column> bindTargetColumns(OlapTable table, List<String> colsName) {
                 }).collect(Collectors.toList());
     }
 
+    private void maybeFallBackAggStateTypeColumn(Column column, ConnectContext ctx) {
```

Comment:
i think u should add aggstate type into `org.apache.doris.nereids.rules.rewrite.CheckDataTypes#UNSUPPORTED_TYPE`